tardis/io/model/tests/test_model_io.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+
+from astropy import units as u
+
+from pathlib import Path
+
+from tardis.io.model import read_stella_model
+
+MODEL_DATA_PATH = Path(__file__).parent / "data"
```

Comment:
Not a huge fan of using __file__, why not just `..`? Or is this more platform agnostic?

Reply:
there is no way of guaranteeing in what directory you are when the tests run. This gets you definitely the location of this file absolutely.